test/public/runs/overview.test.js

```diff
@@ -1099,19 +1101,33 @@ module.exports = () => {
         expect(popoverContent).to.equal('Duration based on o2 start AND stop because of missing trigger information');
     });
 
-    it('should successfully display links to infologger and QC GUI', async () => {
+    it('should successfully display links to infologger, QC GUI and ECS', async () => {
+        await runService.create({ runNumber: 1000, timeTrgStart: new Date(), environmentId: 'CmCvjNbg' });
```

Comment:
Same thing, I think this would be better at the start of the test suite

Reply:
applied